### PR TITLE
Ref/update remote

### PR DESCRIPTION
**Pull request by matiu**

### CODE REVIEW

**Comment by almightyquee** [comment]
Pmo

**Comment by almightyquee** [comment]
Text my phone 6018808410

File: src/providers/profile/profile.ts [hunk 1]
```diff
@@ -1446,22 +1459,35 @@ export class ProfileProvider {
...
 
-  private _createTokenWallet(ethWallet, token) {
-    if (_.isString(token)) {
-      let tokens = this.currencyProvider.getAvailableTokens();
-      token = tokens.find(x => x.symbol == token);
-    }
-    const tokenCredentials = ethWallet.credentials.getTokenCredentials(token);
+  private _createTokenWallet(ethWallet, tokenObj) {
+    this.logger.debug(
+      `Creating token wallet ${tokenObj.name} for ${ethWallet.id}:`
```
Comment (cmgustavo) [inline src/providers/profile/profile.ts]: Getting this in the console.log: 
``` 
[debug] Creating token wallet USD Coin for undefined:
```

Comment (almightyquee) [reply src/providers/profile/profile.ts]: Ok

**Pull request merged**